tests/testthat/test-sec-axis.R

```diff
@@ -0,0 +1,45 @@
+context("sec-axis")
+
+x <- exp(seq(log(0.001), log(1000), length.out = 100))
+foo <- data.frame(
+  x = x,
+  y = x/(1+x)
+)
+
+test_that("dup_axis() works", {
+  vdiffr::expect_doppelganger(
+    "dup_axis",
+    ggplot(foo, aes(x, y)) +
+      geom_point() +
+      scale_x_continuous(name = "Unit A",
+                         sec.axis = dup_axis())
+  )
+})
+
+test_that("custom breaks works", {
+  vdiffr::expect_doppelganger(
```

Comment:
I think it's good to have one visual test to ensure the whole process basically works correctly. And then regular tests for everything else.